app/components/HowCanIHelp/CatgoryCarousel.tsx

```diff
@@ -8,7 +8,12 @@ import './category-carousel.css'
 
 const PER_BOX = 320
 
-const CategoryCarousel = ({title, category}: {title: string; category: PageId}) => {
+interface CategoryCarouselProps {
```

Comment:
can you do `type CategoryCarouselProps = {`? For consistency with the other parts of the code